### PR TITLE
Prepare sdk 3.1.0 and CLI 1.8.0 releases + phone-number-privacy--** bump

### PR DESCRIPTION
**Pull request by aaronmgdr**
### Description

1. bumps SDK and CLI versions

2. change phone-number-privacy packages to depend on caret (ie any minor version greater than specified) rather than locked to one verison of sdks

3. have sdks that use phone-number-privacy packages depend on caret version of them. 

This means sdks can have version changed without needing to upgrade the phone-number packages too going forward. 

4. CHANGE the sdk deploy script to keep `^` and `~` if they exist before version 

5. remove `.npmignore` from `cli` package as it was not being used. instead use `package.json`s `files` array

### CODE REVIEW

**Review by lvpeschke — APPROVED** [review]
LGTM

**Review by soloseng — APPROVED** [review]
🚀

**Comment by aaronmgdr** [comment]
plan to merge this next week after taking the releases out of beta

File: packages/phone-number-privacy/combiner/package.json [hunk 1]
```diff
@@ -28,10 +28,10 @@
     "test:e2e:alfajores": "FLAKEY=true CONTEXT_NAME=alfajores yarn test:e2e"
   },
   "dependencies": {
-    "@celo/contractkit": "3.0.2-dev",
-    "@celo/phone-number-privacy-common": "2.0.2-dev",
-    "@celo/identity": "3.0.2-dev",
-    "@celo/encrypted-backup": "3.0.2-dev",
+    "@celo/contractkit": "^3.1.0",
+    "@celo/phone-number-privacy-common": "^2.0.2",
+    "@celo/identity": "^3.1.0",
```
Comment (alecps) [inline packages/phone-number-privacy/combiner/package.json]: I'd like to better understand why this is necessary before making this change

**Review by alecps — CHANGES_REQUESTED** [review]

**Review by alecps — APPROVED** [review]
spoke with Aaron and these version changes make sense to me 👍

**Pull request merged**